src/figma/tokens.json

```
@@ -559,231 +559,231 @@
           "type": "color"
         }
       },
       "icon": {
         "default": {
-          "value": "hsla(210, 18%, 96%, 1)",
+          "value": "#F2F4F6",
           "description": "(grey040: #F2F4F6) icon.default should be used for all icons used as CTAs that are not a primary action on background.default or background.alternative",
           "type": "color"
         },
         "muted": {
-          "value": "hsla(213, 8%, 55%, 1)",
+          "value": "#9FA6AE",
           "description": "(grey300: #9FA6AE) icon.muted should be used for all low priority icon used on background.default or background.alternative. These could also include placeholder or inactive icons. It will also not meet AAA or AA accessibility standards.",
           "type": "color"
         }
       },
       "border": {
         "default": {
-          "value": "hsla(213, 8%, 55%, 1)",
+          "value": "#848C96",
           "description": "(HEX: #848C96) border.default should be used for all neutral component border colors such as cards or inputs in default state.",
           "type": "color"
         },
         "muted": {
-          "value": "hsla(213, 9%, 25%, 1)",
+          "value": "#3B4046",
           "description": "(grey700: #3B4046) border.muted should be used for any borders with low contrast or subtle definition like dividers",
           "type": "color"
         }
       },
       "overlay": {
         "default": {
-          "value": "hsla(0, 0%, 100%, .4)",
+          "value": "#FFFFFF40",
           "description": "(white000: #FFFFFF 40% opacity) overlay.default should be used for all general overlay backgrounds",
           "type": "color"
         },
         "inverse": {
-          "value": "hsla(0, 0%, 99%, 1)",
+          "value": "#FCFCFC",
           "description": "(white010: #FCFCFC) overlay.inverse should be used only as the foreground element on top of overlay.default used for text or icons",
           "type": "color"
         }
       },
       "primary": {
         "default": {
-          "value": "hsla(205, 98%, 53%, 1)",
+          "value": "#1098FC",
           "description": "(blue400: #1098FC) primary.default should be used for all primary action components such as buttons or links.",
           "type": "color"
         },
         "alternative": {
-          "value": "hsla(205, 97%, 63%, 1)",
+          "value": "#43AEFC",
           "description": "(blue300: #43AEFC) primary.alternative should be used as an alternative to primary.default for things such as hover states",
           "type": "color"
         },
         "muted": {
-          "value": "hsla(205, 98%, 53%, .15)",
+          "value": "#1098FC10",
           "description": "(blue400: #1098FC 10% opacity) primary.muted is a very low contrasting primary variant for things such as alert backgrounds. primary.muted and primary.inverse should not be used together in a foreground and background combination",
           "type": "color"
```

Oh shoot. I lead you astray. I forgot we tested 10% opacity in darkmode and it didn't quite stack up as well as it did in light mode. I think muted should be 15%

```suggestion
          "value": "#1098FC15",
          "description": "(blue400: #1098FC 15% opacity) primary.muted is a very low contrasting primary variant for things such as alert backgrounds. primary.muted and primary.inverse should not be used together in a foreground and background combination",
          "type": "color"
```